packages/cln_plugin/.gitignore

```diff
@@ -8,3 +8,6 @@ build/
 # Omit committing pubspec.lock for library packages; see
 # https://dart.dev/guides/libraries/private-files#pubspeclock.
 pubspec.lock
+
+# Omit commiting executable file, will remove before merging
+/example/cln_plugin_example.exe
```

Comment:
I need to remove this, now the cln is generate with `make`

Reply:
Sure :)